jadx-gui/src/main/java/jadx/gui/ui/codearea/FridaAction.java

```diff
@@ -78,10 +80,18 @@ private String generateMethodSnippet(JMethod jMth) {
 		MethodNode mth = jMth.getJavaMethod().getMethodNode();
 		MethodInfo methodInfo = mth.getMethodInfo();
 		String methodName;
+		String newMethodName;
 		if (methodInfo.isConstructor()) {
 			methodName = "$init";
+			newMethodName = methodName;
 		} else {
 			methodName = StringEscapeUtils.escapeEcmaScript(methodInfo.getName());
+			newMethodName = methodName;
```

Comment:
Just use `methodInfo.getAlias()`
```suggestion
			newMethodName = StringEscapeUtils.escapeEcmaScript(methodInfo.getAlias());
```

Reply:
Oh ye you are correct, I missied this alias proprety so I retrived it directly from the renames list. I will change it and commit :)